codespell_lib/data/dictionary.txt

```diff
@@ -25132,10 +25386,12 @@ watn->want
 wavelenght->wavelength
 wavelenghts->wavelengths
 wavelnes->wavelines
+wayoints->waypoints
```

Comment:
Would you mind also adding `wayoint->waypoint` ?

Reply:
Added wayoint->waypoint